docs/UserGuide.md

```diff
@@ -1,125 +1,151 @@
 # RMD User Guide:
 
-## Pre-requires
+## Prerequisite
 
-Please read the [requirements][requirements].
+To use RMD, the users need to meet several requirements for both hardware and
+software. Please read the [this doc](../docs/Prerequisite.md) for more details.
 
-[requirements]: https://github.com/intel/rmd/blob/master/docs/PreRequirment.md
+## Installation
 
-## RMD processes
+Currently, RMD supports only executable binary file downloading, and the users
+need to install it to system path manually.
 
-For security considerations, RMD daemon runs the RESTAPI service (a spawned
-child) as a normal unix user 'rmd' process and performs privileged operations
-(PAM authentication or resctrl) as 'root' unix user.
+### Download
 
-Ensure the following libraries are installed on your target system to use PAM
-Debian
-```
-sudo apt-get install openssl libpam0g-dev db-util
-```
-Redhat
-```
-sudo dnf install openssl pam-devel db4-utils
-```
-
-Clients are validated against host unix credentials or credentials setup in a
-Berkeley database file.
-To setup users in Berkeley database use the bash script:
-
-```
-sudo ./scripts/setup_rmd_users.sh
-```
-Note: Only root user can setup or access users in Berkeley database.
-
-Launch RMD by supplying configuration directory using the command
+The RMD executable binary files are hosted in github.com infrastructure, and
+you can find the download links from [release notes](TODO).
 
-```
-sudo ./rmd --conf-dir ./etc/rmd
-```
+### Setup configuration file
 
-`RMD can be launched in debug mode that exposes RESTAPI service on HTTP.
-By default it is launched on port 8081. (http://127.0.0.1:8081)`
+Most of the configuration files are in [TOML](https://github.com/toml-lang/toml) format.
+Users can create their own configuration files by referring the following
+sample. And RMD also provides the script(cmd/gen_conf.go) to generate it
+by probing the local host platform capabilities for proper CachePool settings.
 
-## Query cache information on the host
+<TODO: need to install the etc/rmd/* to system manually?>
```

Comment:
Yep, if this guide is for enduser (who downloads the binary), he/she should install etc/rmd/* to /etc/rmd/ manually.